codegen/src/shuttle_main/mod.rs

```diff
@@ -14,9 +14,15 @@ pub(crate) fn r#impl(_attr: TokenStream, item: TokenStream) -> TokenStream {
     let loader = Loader::from_item_fn(&mut fn_decl);
 
     quote! {
-        #[tokio::main]
-        async fn main() {
-            shuttle_runtime::start(loader).await;
+        fn main() {
+            // manual expansion of #[tokio::main]
+            ::shuttle_runtime::tokio::runtime::Builder::new_multi_thread()
```

Comment:
Can we turn off some tokio defaults because of this? I'm thinking of turning off the `macros` feature to reduce the dependencies on users.

Reply:
We can also do it in a separate PR

Reply:
It seems like other dependencies are enabling it. It's a messy task.